com.unity.uiwidgets/Runtime/widgets/app.cs

```diff
@@ -281,40 +283,7 @@ public class WidgetsApp : StatefulWidget {
         }
     }
 
-    public class WindowProvider : InheritedWidget {
```

Comment:
this change is big, please check all the samples in the WidgetsSample folder, ensuring the correctness

Reply:
This class is completely deleted in the flutter 1.17.5.